feature-libs/asm/components/customer-list/customer-list.component.ts

```diff
@@ -217,16 +217,30 @@ export class CustomerListComponent implements OnInit, OnDestroy {
 
       this.asmCustomerListFacade.customerListCustomersSearch(options);
     }
-    this.customerListConfig?.columns?.forEach((item) => {
+    this.updateCustomerListColumns();
+  }
+
+  private updateCustomerListColumns(): void {
+    const columns = this.customerListConfig?.columns || [];
+
+    for (const column of columns) {
       if (
-        item.headerLocalizationKey === 'asm.customerList.tableHeader.account' ||
-        item.headerLocalizationKey === 'hideAccount'
+        column.headerLocalizationKey ===
+          'asm.customerList.tableHeader.account' ||
+        column.headerLocalizationKey === 'hideAccount'
       ) {
-        item.headerLocalizationKey = this.enableAsmB2bCustomerList
+        column.headerLocalizationKey = this.enableAsmB2bCustomerList
           ? 'asm.customerList.tableHeader.account'
           : 'hideAccount';
       }
-    });
+      if (
+        column.headerLocalizationKey === 'asm.customerList.tableHeader.cart'
+      ) {
+        column.headerLocalizationKey = this.featureConfig?.isLevel('6.1')
+          ? column.headerLocalizationKey
+          : 'hideAccount';
```

Comment:
Please don't use 'hideAccount' anymore. Since we don't only hide account column, let's change it to other string, maybe like 'hideHeader'.

Reply:
Done